python3/agents/shared/utils/constants.py

```diff
@@ -56,5 +56,7 @@
     # special rewards
     'enemy': -2,
     'pinch': -2,  # articulation point
-    'wall': -10
+    'wall': -10,
```

Comment:
No need to add destructible plant and hazard zone here because I the block destroy strat is the only one using it